tests/integration/api_swarm_test.py

```diff
@@ -131,7 +131,7 @@ def test_update_swarm_name(self):
         assert self.init_swarm()
```

Comment:
hm, perhaps this test should be removed altogether if this is testing that the name can be updated (let me know if it should)

Reply:
Yup, test should be removed altogether if naming isn't supported.

Reply:
I'll update the PR, thanks!